ebill_paynet/tests/test_ebill_paynet.py

```diff
@@ -98,7 +114,7 @@ def test_takeShipment(self):
         attachment = self.env["ir.attachment"].search(
             [["res_model", "=", "res.country"], ["res_id", "=", ch.id]]
         )
-        attachment = self.env.ref("mail.msg_discus4_attach1")
+        attachment = self.env.ref("mass_mailing.mass_mail_attach_1")
```

Comment:
It's a bummer to add a new dependency `mass_mailing` to this module only for the test.
This test only sends any file to the service and checks that it receives a shipment name.
Could you try to use another file as to not add the dependency on `mass_mailing`. Either one in Odoo or one in `ebill_paynet/tests/examples`

Reply:
well noted, I'll update it